packages/flutter/lib/src/widgets/notification_listener.dart

```diff
@@ -58,15 +58,36 @@ abstract class Notification {
   /// Subclasses can override this to apply additional filtering or to update
   /// the notification as it is bubbled (for example, increasing a `depth` field
   /// for each ancestor of a particular type).
+  ///
+  /// This functionality was deprecated after v1.11.0 because of poor performance.
```

Comment:
Update the docs above to make it very clear that this is not only deprecated, but the feature has already been removed. Overriding this method has no effect whatsover.

Reply:
Actually, I am wondering if we should just remove this altogether then. Seeing this deprecated gives you the false sense that - while this will be removed in the future - it still functions as expected. It just confused myself when I took another look at this PR (see my top-level comment).

The code snippet should then go into a migration guide on our website that people will hopefully find when they get invalid override warnings.

Reply:
OK, that sounds good. I guess its not a compile error to override stuff that isn't there, just a warning